This patch fixes the apparmor parser tst Makefile to not use popd/pushd
bashisms which were in place to get around PWD not getting set
correctly. Instead, fix the simple.pl script to call pwd directly.
Based on feedback from PLD/Arkadiusz Miskiewicz <arekm@maven.pl>.
---
 parser/tst/Makefile  |    2 +-
 parser/tst/simple.pl |    8 +++++---
 2 files changed, 6 insertions(+), 4 deletions(-)

Index: trunk-forge/parser/tst/Makefile
===================================================================
--- trunk-forge.orig/parser/tst/Makefile
+++ trunk-forge/parser/tst/Makefile
@@ -12,7 +12,7 @@ all: tests
 
 .PHONY: tests
 tests: ../apparmor_parser
-	$(Q)pushd . && ${PROVE} ${PROVE_ARG} ${TESTS} && popd
+	$(Q)${PROVE} ${PROVE_ARG} ${TESTS}
 
 ../apparmor_parser:
 	make -C .. apparmor_parser
Index: trunk-forge/parser/tst/simple.pl
===================================================================
--- trunk-forge.orig/parser/tst/simple.pl
+++ trunk-forge/parser/tst/simple.pl
@@ -13,6 +13,8 @@ $config{'parser'} = "/sbin/subdomain_par
 $config{'profiledir'} = "./simple_tests/";
 
 my $help;
+my $pwd = `pwd`;
+chomp($pwd);
 
 GetOptions(
   "help|h" => \$help,
@@ -32,13 +34,13 @@ read_config();
 # Override config file profile location when passed on command line
 if (@ARGV >= 1) {
   $config{'profiledir'} = shift;
-} 
+}
 
 if ($config{'profiledir'} =~ /^\//) {
   $config{'includedir'} = $config{'profiledir'};
 } else {
-  $config{'includedir'} = "$ENV{'PWD'}/$config{'profiledir'}";
-} 
+  $config{'includedir'} = "$pwd/$config{'profiledir'}";
+}
 
 sub read_config {
   my $which;
